### PR TITLE
Show correct description for empty Connected Sites modal

### PR DESCRIPTION
**Pull request by whymarrh**
This PR updates the description shown for an empty `ConnectedSites` modal

<details>
<summary>Screenshots</summary>
Screen Shot 2020-04-07 at 21 34 50
Screen Shot 2020-04-07 at 21 34 42
</details>

### CODE REVIEW

File: ui/app/pages/connected-sites/connected-sites.container.js [hunk 1]
```diff
@@ -13,6 +14,7 @@ const mapStateToProps = (state) => {
   const { openMetaMaskTabs } = state.appState
   const { title, url, id } = state.activeTab
   const permittedAccounts = getPermittedAccountsForCurrentTab(state)
+  const connectedDomains = getRenderablePermissionsDomains(state)
```
Comment (Gudahtt) [inline ui/app/pages/connected-sites/connected-sites.container.js:17]: This selector seems a bit overkill - none of the info it's collecting is used. That means changes to that data will result in a new value here, and then a new render, despite there being no change in the number of connected domains.

I don't see any other selectors with this info though :thinking: Maybe this calls for a new selector?

File: app/_locales/en/messages.json [hunk 2]
```diff
@@ -5,6 +5,9 @@
   "connectedSitesDescription": {
     "message": "$1 is connected to these sites. They can view your account address."
   },
+  "connectedSitesEmptyDescription": {
+    "message": "$1 is not connected to any sites."
```
Comment (Gudahtt) [inline app/_locales/en/messages.json:9]: Ideally any message with a `$1` would have an accompanying description that explains what the `$1` is - to help translators understand the context. It looks like I missed this with the last PR (the message above this one also lacks a description).

**Review by Gudahtt — CHANGES_REQUESTED** [review]

File: ui/app/pages/connected-sites/connected-sites.container.js (hunk 1 above)
Comment (whymarrh) [reply ui/app/pages/connected-sites/connected-sites.container.js:17]: What if I just passed connectedDomains through to the list component instead?

File: app/_locales/en/messages.json (hunk 2 above)
Comment (whymarrh) [reply app/_locales/en/messages.json:9]: I'll add a description for this one, yeah

File: ui/app/pages/connected-sites/connected-sites.container.js (hunk 1 above)
Comment (Gudahtt) [reply ui/app/pages/connected-sites/connected-sites.container.js:17]: Oh, huh, I hadn't realized it was used by the child component. Yeah that makes a lot of sense!

Comment (whymarrh) [reply ui/app/pages/connected-sites/connected-sites.container.js:17]: Updated to pass it through

File: app/_locales/en/messages.json (hunk 2 above)
Comment (whymarrh) [reply app/_locales/en/messages.json]: Updated and created #8303 as well

**Comment by whymarrh** [comment]
I added screenshots to the OP

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**